Utilities/bootstrap

```diff
@@ -462,7 +478,7 @@ def install_binary(args, binary, dest_dir):
 # Build functions
 # -----------------------------------------------------------
 
-def build_with_cmake(args, cmake_args, source_path, build_dir, targets=[]):
+def build_with_cmake(args, cmake_args, source_path, build_dir):
```

Comment:
Now that the only use of this `targets` parameter was removed below, as the PD libraries no longer need to be built by CMake, I removed it altogether.